tests/Oryx.Integration.Tests/Node/NodeDynamicInstallationTest.cs

```diff
@@ -33,10 +33,10 @@ public async Task CanBuildAndRunNode14AppUsingDynamicInstallationOfRuntimeInRunt
         [Fact]
         [Trait("category", "node-16")]
         [Trait("build-image", "github-actions-debian-buster")]
-        public async Task CanBuildAndRunNode16AppUsingDynamicInstallationOfRuntimeInRuntimeImageAsync()
-        {
-            await CanBuildAndRunAppUsingDynamicInstallationOfRuntimeInRuntimeImageAsync(NodeVersions.Node16Version);
-        }
+        // public async Task CanBuildAndRunNode16AppUsingDynamicInstallationOfRuntimeInRuntimeImageAsync()
+        // {
+        //     await CanBuildAndRunAppUsingDynamicInstallationOfRuntimeInRuntimeImageAsync(NodeVersions.Node16Version);
+        // }
```

Comment:
lets add a todo to enable it back once we have changes relate to getting the latest node 16 version from variable group for all the tests as well